cmd/postgres-operator/main.go

```diff
@@ -87,10 +90,29 @@ func main() {
 // runtime manager.
 func addControllersToManager(mgr manager.Manager) error {
 	r := &postgrescluster.Reconciler{
-		Client:   mgr.GetClient(),
-		Owner:    postgrescluster.ControllerName,
-		Recorder: mgr.GetEventRecorderFor(postgrescluster.ControllerName),
-		Tracer:   otel.Tracer(postgrescluster.ControllerName),
+		Client:      mgr.GetClient(),
+		Owner:       postgrescluster.ControllerName,
+		Recorder:    mgr.GetEventRecorderFor(postgrescluster.ControllerName),
+		Tracer:      otel.Tracer(postgrescluster.ControllerName),
+		IsOpenshift: isOpenshift(mgr.GetConfig()),
```

Comment:
⛏️ OpenShift has a capital `S`, so we should do the same.

```suggestion
		IsOpenShift: isOpenShift(mgr.GetConfig()),
```